src/settings-ui/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -495,7 +495,10 @@
     <value>Excluded apps</value>
   </data>
   <data name="ShortcutGuide_ExcludedApps_TextBoxControl.Header" xml:space="preserve">
-    <value>To exclude an application add its name here (one per line).</value>
+    <value>To exclude an application add its name here (one per line). Shortcut Guide won't react to Windows key for these apps.</value>

```

Comment:
We need to rephrase this, since we are not "excluding" an application, we are preventing the SG to interfere with that application.
Something like "Turn off Shortcut Guide when these applications have focus. Add one application name per line." 
Any better suggestion is welcomed.

Reply:
Then we should change group haeder to. "Exculde Apps" makes no sense zhen. Suggestion is `Disable for Apps` or something similar.